_world_speakers/2024/speakers/yukihiro-matz.md

```diff
@@ -4,7 +4,7 @@ first_name: Yukihiro "Matz"
 last_name: Matsumoto
 image_path: /assets/world/2024/images/speakers/m-matsumoto.jpg
 role: Creator of Ruby
-company: 
+company: false
```

Comment:
I was having trouble getting rid of a comma because Yukihiro had no company tag, so I set this as false and am checking for it in the speaker cards + the session cards

Reply:
Could you please check and make sure false isn't showing for the home screens speaker cards? I couldn't get to this on my local

Reply:
Will do!